client/modules/core/helpers/layout.js

```diff
@@ -19,6 +19,7 @@ Template.registerHelper("reactionTemplate", function (options) {
   const shopId = options.hash.shopId || Reaction.getShopId();
   // get shop info, defaults to current
   const Shop = Collections.Shops.findOne(shopId);
+  const defaultRoles = Collections.Groups.findOne({ slug: "customer", shopId }).permissions;
```

Comment:
This helper is only used in the checkout right now it seems, and even if we load the checkout directly default roles are ready, so I think we're good to go here.